server/graphql/v2/mutation/EmojiReactionMutations.js

```diff
@@ -0,0 +1,130 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { mustBeLoggedInTo } from '../../../lib/auth';
+import models from '../../../models';
+import { canComment } from '../../common/expenses';
+import { Forbidden, NotFound, Unauthorized, ValidationFailed } from '../../errors';
+import { idDecode, IDENTIFIER_TYPES } from '../identifiers';
+import { CommentReferenceInput } from '../input/CommentReferenceInput';
+import { UpdateReferenceInput } from '../input/UpdateReferenceInput';
+import { EmojiReactionsOutput } from '../object/EmojiReactionsOutput';
+
+const emojiReactionMutations = {
+  addEmojiReaction: {
+    type: new GraphQLNonNull(EmojiReactionsOutput),
```

Comment:
Since `EmojiReactionsOutput` is very specific to these mutations, I would keep the object declaration in the same file. The type name can also be more explicit, I can suggest `EmojiReactionResponse`